src/integritysetup/integritysetup-generator.c

```diff
@@ -85,6 +85,7 @@ static int create_disk(
                 "DefaultDependencies=no\n"
                 "IgnoreOnIsolate=true\n"
                 "After=integritysetup-pre.target systemd-udevd-kernel.socket\n"
+                "After=modprobe@dm_integrity.service\n"
```

Comment:
hmm, why do we have to load the manually? normally module auto-loading by the DM layer should suffice? why not here?

Reply:
Hmm. Not sure. But the reporter said that the original issue can be fixed by adding the module to modules-load.d.